DEBUG - 2025-01-19 02:00:16 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\FileHandler' driver.
DEBUG - 2025-01-19 02:00:25 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\FileHandler' driver.
ERROR - 2025-01-19 02:00:30 --> Error connecting to the database: mysqli_sql_exception: No connection could be made because the target machine actively refused it in C:\xampp\htdocs\CRUDCI4\system\Database\MySQLi\Connection.php:186
Stack trace:
#0 C:\xampp\htdocs\CRUDCI4\system\Database\MySQLi\Connection.php(186): mysqli->real_connect('localhost', 'root', Object(SensitiveParameterValue), 'sistem_sekolah', 3306, '', 0)
#1 C:\xampp\htdocs\CRUDCI4\system\Database\BaseConnection.php(421): CodeIgniter\Database\MySQLi\Connection->connect(false)
#2 C:\xampp\htdocs\CRUDCI4\system\Database\BaseConnection.php(616): CodeIgniter\Database\BaseConnection->initialize()
#3 C:\xampp\htdocs\CRUDCI4\system\Database\BaseBuilder.php(1644): CodeIgniter\Database\BaseConnection->query('SELECT 1\nFROM `...', Array, false)
#4 C:\xampp\htdocs\CRUDCI4\system\Validation\StrictRules\Rules.php(232): CodeIgniter\Database\BaseBuilder->get()
#5 C:\xampp\htdocs\CRUDCI4\system\Validation\Validation.php(338): CodeIgniter\Validation\StrictRules\Rules->is_unique('manca', 'username', Array, NULL, 'username')
#6 C:\xampp\htdocs\CRUDCI4\system\Validation\Validation.php(203): CodeIgniter\Validation\Validation->processRules('username', 'username', 'manca', Array, Array, 'username')
#7 C:\xampp\htdocs\CRUDCI4\system\Controller.php(138): CodeIgniter\Validation\Validation->run()
#8 C:\xampp\htdocs\CRUDCI4\app\Controllers\Auth.php(32): CodeIgniter\Controller->validate(Array)
#9 C:\xampp\htdocs\CRUDCI4\system\CodeIgniter.php(933): App\Controllers\Auth->processRegister()
#10 C:\xampp\htdocs\CRUDCI4\system\CodeIgniter.php(509): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Auth))
#11 C:\xampp\htdocs\CRUDCI4\system\CodeIgniter.php(355): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#12 C:\xampp\htdocs\CRUDCI4\system\Boot.php(334): CodeIgniter\CodeIgniter->run()
#13 C:\xampp\htdocs\CRUDCI4\system\Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
#14 C:\xampp\htdocs\CRUDCI4\public\index.php(56): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
#15 C:\xampp\htdocs\CRUDCI4\system\rewrite.php(44): require_once('C:\\xampp\\htdocs...')
#16 {main}

Next CodeIgniter\Database\Exceptions\DatabaseException: No connection could be made because the target machine actively refused it in C:\xampp\htdocs\CRUDCI4\system\Database\MySQLi\Connection.php:231
Stack trace:
#0 C:\xampp\htdocs\CRUDCI4\system\Database\BaseConnection.php(421): CodeIgniter\Database\MySQLi\Connection->connect(false)
#1 C:\xampp\htdocs\CRUDCI4\system\Database\BaseConnection.php(616): CodeIgniter\Database\BaseConnection->initialize()
#2 C:\xampp\htdocs\CRUDCI4\system\Database\BaseBuilder.php(1644): CodeIgniter\Database\BaseConnection->query('SELECT 1\nFROM `...', Array, false)
#3 C:\xampp\htdocs\CRUDCI4\system\Validation\StrictRules\Rules.php(232): CodeIgniter\Database\BaseBuilder->get()
#4 C:\xampp\htdocs\CRUDCI4\system\Validation\Validation.php(338): CodeIgniter\Validation\StrictRules\Rules->is_unique('manca', 'username', Array, NULL, 'username')
#5 C:\xampp\htdocs\CRUDCI4\system\Validation\Validation.php(203): CodeIgniter\Validation\Validation->processRules('username', 'username', 'manca', Array, Array, 'username')
#6 C:\xampp\htdocs\CRUDCI4\system\Controller.php(138): CodeIgniter\Validation\Validation->run()
#7 C:\xampp\htdocs\CRUDCI4\app\Controllers\Auth.php(32): CodeIgniter\Controller->validate(Array)
#8 C:\xampp\htdocs\CRUDCI4\system\CodeIgniter.php(933): App\Controllers\Auth->processRegister()
#9 C:\xampp\htdocs\CRUDCI4\system\CodeIgniter.php(509): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Auth))
#10 C:\xampp\htdocs\CRUDCI4\system\CodeIgniter.php(355): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#11 C:\xampp\htdocs\CRUDCI4\system\Boot.php(334): CodeIgniter\CodeIgniter->run()
#12 C:\xampp\htdocs\CRUDCI4\system\Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
#13 C:\xampp\htdocs\CRUDCI4\public\index.php(56): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
#14 C:\xampp\htdocs\CRUDCI4\system\rewrite.php(44): require_once('C:\\xampp\\htdocs...')
#15 {main}
CRITICAL - 2025-01-19 02:00:30 --> CodeIgniter\Database\Exceptions\DatabaseException: Unable to connect to the database.
Main connection [MySQLi]: No connection could be made because the target machine actively refused it
[Method: POST, Route: register]
in SYSTEMPATH\Database\BaseConnection.php on line 467.
 1 SYSTEMPATH\Database\BaseConnection.php(616): CodeIgniter\Database\BaseConnection->initialize()
 2 SYSTEMPATH\Database\BaseBuilder.php(1644): CodeIgniter\Database\BaseConnection->query('SELECT 1
FROM `users`
WHERE `username` = :username:
 LIMIT 1', [...], false)
 3 SYSTEMPATH\Validation\StrictRules\Rules.php(232): CodeIgniter\Database\BaseBuilder->get()
 4 SYSTEMPATH\Validation\Validation.php(338): CodeIgniter\Validation\StrictRules\Rules->is_unique('manca', 'username', [...], null, 'username')
 5 SYSTEMPATH\Validation\Validation.php(203): CodeIgniter\Validation\Validation->processRules('username', 'username', 'manca', [...], [...], 'username')
 6 SYSTEMPATH\Controller.php(138): CodeIgniter\Validation\Validation->run()
 7 APPPATH\Controllers\Auth.php(32): CodeIgniter\Controller->validate([...])
 8 SYSTEMPATH\CodeIgniter.php(933): App\Controllers\Auth->processRegister()
 9 SYSTEMPATH\CodeIgniter.php(509): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Auth))
10 SYSTEMPATH\CodeIgniter.php(355): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
11 SYSTEMPATH\Boot.php(334): CodeIgniter\CodeIgniter->run()
12 SYSTEMPATH\Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
13 FCPATH\index.php(56): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
14 SYSTEMPATH\rewrite.php(44): require_once('C:\\xampp\\htdocs\\CRUDCI4\\public\\index.php')
DEBUG - 2025-01-19 02:01:03 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\FileHandler' driver.
DEBUG - 2025-01-19 02:01:10 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\FileHandler' driver.
DEBUG - 2025-01-19 02:01:10 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\FileHandler' driver.
DEBUG - 2025-01-19 02:01:26 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\FileHandler' driver.
DEBUG - 2025-01-19 02:01:27 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\FileHandler' driver.
DEBUG - 2025-01-19 02:01:33 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\FileHandler' driver.
DEBUG - 2025-01-19 02:01:33 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\FileHandler' driver.
